src/TransloaditClient.js

```diff
@@ -563,7 +691,49 @@ class TransloaditClient {
     })
 
     if (method === 'post' || method === 'put' || method === 'del') {
-      this._appendForm(req, opts.params, opts.fields)
+      const extraData = Object.assign(
+        {tus_num_expected_upload_files: opts.tus_num_expected_upload_files},
+        opts.fields
+      )
+      this._appendForm(req, opts.params, extraData)
+    }
+  }
+
+  _sendTusRequest (opts, cb, onProgress) {
+    let uploadsDone = 0
+    const streamLabels = Object.keys(this._tus_streams)
+    const tlClient = this
+    onProgress = onProgress || (() => {})
+    for (const label of streamLabels) {
+      const file = this._tus_streams[label]
+      const filename = file.path ? path.basename(file.path) : label
+      const tusUpload = new tus.Upload(file, {
+        endpoint: opts.tus_url,
+        resume: true,
+        metadata: {
+          assembly_url: opts.assembly_ssl_url,
+          fieldname: label,
+          filename
+        },
+        uploadSize: fs.statSync(file.path).size, // todo this will fail for streams without path
```

Comment:
Are there plans to add support for streams in this PR?

Reply:
@Acconut not in this PR, but I added a check so that if we can not detect the stream size, it will fallback to multipart upload